components/expenses/ExpenseList.jsx

```diff
@@ -0,0 +1,87 @@
+import { View, Text, StyleSheet } from "react-native";
+import React from "react";
+import ExpenseCard from "./ExpenseCard";
+import ExpenseAdder from "./ExpenseAdder";
+
+const data = {
+  expenses: [
+    {
+      id: 1,
+      amount: 8.25,
+      merchant: "Starbucks",
+      category: "Food/Beverage",
+      date: null,
+      receipt: null,
+      location: null,
+      accountSelected: null,
+    },
+    {
+      id: 2,
+      amount: 8.25,
+      merchant: "Starbucks",
+      category: "Food/Beverage",
+      date: null,
+      receipt: null,
+      location: null,
+      accountSelected: null,
+    },
+    {
+      id: 3,
+      amount: 8.25,
+      merchant: "Starbucks",
+      category: "Food/Beverage",
+      date: null,
+      receipt: null,
+      location: null,
+      accountSelected: null,
+    },
+    {
+      id: 4,
+      amount: 8.25,
+      merchant: "Starbucks",
+      category: "Food/Beverage",
+      date: null,
+      receipt: null,
+      location: null,
+      accountSelected: null,
+    },
+    {
+      id: 5,
+      amount: 8.25,
+      merchant: "Starbucks",
+      category: "Food/Beverage",
+      date: null,
+      receipt: null,
+      location: null,
+      accountSelected: null,
+    },
+  ],
+};
+
+export default function ExpenseList() {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>ExpenseList</Text>
+      {data.expenses.map((exp) => {
+        return <ExpenseCard key={exp.id} exp={exp} />;
+      })}
+      {/* <ExpenseAdder /> */}
+    </View>
+  );
+}
```

Comment:
I have found that FlatList is a better option to using map() in terms of performance, depending on how large the dataset is. It might be a good idea to implement it now, so it would be able to handle large datasets in the future, as our app grows in popularity.